### ISSUE TITLE
readthedocs API changes

### ISSUE THREAD

**Issue by jeremymanning** [ContextLab/hypertools#159]
I think a few minor changes would help make the API documentation a bit clearer on readthedocs:

- Make a note in `hypertools.tools.describe_pca` that the function is depreciated (replaced by `hypertools.describe`
- Change order of main functions to: plot, analyze, normalize, reduce, align, cluster, describe
- Make a note in `hypertools.tools.procrustes` that the function is depreciated (replaced by `hypertools.align(..., align='procrustes')`
- Does cluster support normalize/reduce/align keywords?  I think it should... (if not, we should open another issue for it; otherwise we should correct the API)
- In plot, analyze, normalize, reduce, align, and cluster we should add more details on how to specify the models.  Specifically, make clear that the user should pass in a string (list possible options) if they want the default parameters.  For advanced users, they should pass a dictionary (and make the scikit-learn "supported models" a hyperlink, so advanced users can explore which parameters are supported)
- For DataGeometry in the high level description, instead of saying "Note: this class should not be called directly, ..." I think we should say "Note: new instances of this class should not be *initialized* directly, ..." (or something along those lines)

**Comment by jeremymanning** [comment]
Status in the 1.0 update (PR #272 https://github.com/ContextLab/hypertools/pull/272): 4 of 6 asks here are already resolved, but not in the way originally requested — `describe_pca` and `procrustes` were removed outright (not deprecated-with-a-note), and `DataGeometry`'s docstring was rewritten to mark it internal-only (stronger than the requested wording tweak). The model-spec string-vs-dict docs ask is genuinely done in `reduce()`/`align()`/`cluster()`. Two items are still open: function listing order, and whether `cluster()` should accept normalize/reduce/align keywords.

**How this could be built on the 1.0 architecture:**
- Reorder imports in `hypertools/__init__.py` (currently `plot, load, analyze, reduce, align, normalize, describe, cluster`) to the requested `plot, analyze, normalize, reduce, align, cluster, describe`, and mirror that order in `docs/api.rst`'s autosummary blocks.
- For the cluster-composition ask: `hypertools/tools/analyze.py`'s `analyze()` already composes `normalize → reduce → align` via `aligner(reducer(normalizer(...)))`, but does not include `cluster`, and `hypertools/cluster/cluster.py`'s `cluster(x, cluster="KMeans", n_clusters=3, format_data=True)` has no normalize/reduce/align kwargs. Two honest paths: (a) add those as pass-through kwargs to `cluster()` that call the same `normalizer`/`reducer`/`aligner` helpers `analyze()` uses before fitting (medium effort, real feature work), or (b) just document in `cluster()`'s docstring that this composition happens via `hyp.plot()`/`hyp.analyze()` today (docs-only, low effort).
- Swap the current prose ("See scikit-learn specific model docs...") for real sphinx cross-references/links to the relevant sklearn pages in `reduce()`/`align()`/`cluster()` docstrings.

**Alternatives / prior art:** n/a — this is a docs-organization ask, not a design question.

Note: parts of this ask are superseded — `describe_pca`/`procrustes` no longer exist to annotate (replaced by `describe()`/`align(..., align='procrustes')`), and `DataGeometry` is already documented as internal-only.

Tagging as **low effort**; leaving open for consideration in a future release.

**Comment by jeremymanning** [comment]
The specifics of the original request aren't relevant anymore; what this task actually entails is: update the sphinx/readthedocs documentation to be current and comprehensive. we need to make sure *all* parts of the user-facing toolbox are covered, with plenty of nice tutorials for showing the many interesting things that can be done with hypertools.

**Comment by jeremymanning** [comment]
Addressed in PR #272 (HyperTools 1.0), per Jeremy's directive: the specifics of the original request no longer apply — what matters is that the Sphinx/readthedocs documentation is current and comprehensive, covers every user-facing part of the toolbox, and has plenty of tutorials showing what hypertools can do.

`docs/api.rst` now covers every public export in `hypertools/__init__.py`/`hypertools/io/__init__.py`, including previously-missing sections for `manip`, `save`, `impute`, `set_interactive_backend`, `Pipeline`, the 6 autoencoder reducer classes, the 6 gensim vectorizer wrapper classes, and `hyp.io.lsl_stream`. The two orphaned pages (`hypertools.tools.align.rst`, `hypertools.tools.normalize.rst`) were removed. 7 new tutorials/examples were added and real-executed during the Sphinx build (no mocks): `plot_story_trajectories.py`, `plot_datasets_tour.py`, `plot_pipelines_return_model.py`, `plot_autoencoders.py`, `plot_gensim_text.py`, plus notebook extensions to `wikipedia_embeddings.ipynb` and a new `lsl_streaming.ipynb`. A pipeline-order flowchart (`docs/pipeline_order.rst` + `docs/_static/pipeline_order.svg`) documents the canonical stage ordering. Commit `6c3a3733`.

```
cd docs && python -m sphinx -b html . _build/html
```

Build verification (3 iterative full builds while fixing issues): final run is clean of the two target orphan-page warnings and introduces zero new warning categories (`toc.not_included`: 0, `autosummary.import_cycle`: 0, `stub file not found`: 0). Remaining ~50 warning lines are 100% pre-existing, in files this task never touched.

```python
import hypertools as hyp
help(hyp.manip)  # full numpydoc docstring, now indexed in api.rst
```

Full suite: ~1275 tests passing.
